.gitignore

```diff
@@ -1,4 +1,8 @@
 _site
 .sass-cache
 .jekyll-metadata
+.jekyll-cache
 _downloads/**
+.bundle
+vendor
+venv
```

Comment:
`venv` is what you do personally; this is fine, but that means it can go in your global gitignore. It has no place in this gitignore :)
Basically, only add in a local gitignore what the local repository generates; not your local tooling :)

Reply:
I did think about that, but venv is explicitly mentioned in the reader, so...